.travis.yml

```diff
@@ -89,8 +89,7 @@ jobs:
         - travis_wait make test-cmd-url
         - travis_wait make test-cmd-devfile-url
         - travis_wait make test-cmd-debug
-        - travis_wait make test-cmd-devfile-debug
-        - travis_wait make test-cmd-devfile-storage
```

Comment:
Why remove storage from here?

Reply:
Consolidated all devfile to one tests job.

Reply:
Why not remove `travis_wait make test-cmd-devfile-url` from here too?

Reply:
done